Mathlib/Topology/Category/TopCat/Limits/Cofiltered.lean

```diff
@@ -27,7 +27,7 @@ open CategoryTheory
 
 open CategoryTheory.Limits
 
-universe v u w
+universe u v w
```

Comment:
This matches the order in mathlib3. If the re-ordering was deliberate, we should probably either backport or add a porting note.

Reply:
Yes, if you'd like a porting note it could be "Universes reordered so inferable universes come later." But there were many such changes made during the port, and the only fate of such a porting note is that it will be deleted, I think...